boards/adafruit-clue/Makefile.dep

```diff
@@ -14,5 +14,9 @@ ifeq (,$(filter-out stdio_cdc_acm,$(filter stdio_% slipdev_stdio,$(USEMODULE))))
   USEMODULE += stdio_cdc_acm
 endif
 
+ifneq (,$(filter auto_init_screen,$(USEMODULE)))
+  USEMODULE += ili9341
```

Comment:
Why is this pulled in here based on `auto_init_screen` and is it pulled in based on `disp_dev` with the other boards?

Reply:
No particular reason I guess, probably a left-over :)

Reply:
now it's imported when disp_dev is used.